### ISSUE TITLE
Kotlin files in "kotlin" directory are not formatted in Android project

### ISSUE THREAD

**Issue by fairbanksg** [cortinico/ktfmt-gradle#48]
## 🐛 Describe the bug

Since I am writing my Android app in kotlin, I started putting all my source code
in `src/main/kotlin`. However, these files are not formatted by the gradle plugin.

## ⚠️ Current behavior

Only files in `src/main/java` are formatted.

## ✅ Expected behavior

Files in `src/main/kotlin` should be formatted as well.

## 💣 Steps to reproduce

- Create an Android project in Android Studio and configure ktfmt-gradle
- Add a `app/src/main/kotlin` directory
- Add a Kotlin file to it with some incorrect formatting
- Run `./gradlew ktfmtFormat`
- Kotlin file in `app/src/main/kotlin` is not formatted

## 📱 Tech info
 - OS: macOS 11.6.1
 - Library/App version: ktfmt-gradle 0.7.0

**Comment by cortinico** [comment]
> Since I am writing my Android app in kotlin, I started putting all my source code
> in `src/main/kotlin`

I believe this is happening as I'm feeding the `source` of the Ktfmt task with `sourceSet.java.srcDirs`. IIRC this should be fixed by updating the AGP version to 7.x (PRs are more than welcome).

As a workaround you should be able to workaround with:

```
tasks.withType<KtfmtBaseTask>().configureEach {
    source += fileTree("src/main/kotlin")
}
```

**Comment by cortinico** [comment]
This is fixed by #57
Will be available with version 0.9.0
